app/src/main/java/org/wikipedia/page/leadimages/LeadImagesHandler.java

```diff
@@ -168,11 +188,54 @@ private void loadLeadImage(@Nullable String url) {
         if (!isMainPage() && !TextUtils.isEmpty(url) && isLeadImageEnabled()) {
             pageHeaderView.show(isLeadImageEnabled());
             pageHeaderView.loadImage(url);
+            updateCallToAction();
         } else {
             pageHeaderView.loadImage(null);
         }
     }
 
+    private void updateCallToAction() {
+        if ((!Prefs.isSuggestedEditsAddCaptionsUnlocked() && !Prefs.isSuggestedEditsTranslateCaptionsUnlocked()) || getPage() == null) {
+            return;
+        }
+        WikipediaApp app = WikipediaApp.getInstance();
+        String imageName = getPage().getPageProperties().getLeadImageName();
+        String filename = "File:" + imageName;
+
+        imageCaptionDisposable = MediaHelper.INSTANCE.getImageCaptions(filename)
```

Comment:
Not sure about the logic because a local image file should not be able to edit the caption. (e.g.: https://en.wikipedia.org/wiki/Toy_Story_4)

Also, I've noticed that the source description/caption is empty in the "add image caption" screen.

Please see the logic of `updateGalleryDescription` in `GalleryActivity`.